### PR TITLE
Limit Content by User Country

### PR DESCRIPTION
**Pull request by ascii-dev**
## Description
A user should only have access to their country's data.

## Type of change

Please delete options that are not relevant.
- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Review by esirK — APPROVED** [review]
This works. You can also attach the screenshot

File: forms/admin.py [hunk 1]
```diff
@@ -30,7 +30,9 @@ def perms_queryset(self, request, perm):
         return shortcuts.get_objects_for_user(request.user, [perm])
 
     def get_queryset(self, request):
-        return self.perms_queryset(request, 'forms.change_%s' % self.permcode)
+        qs = self.perms_queryset(request, 'forms.change_%s' % self.permcode)
+        qs_filter = qs.filter(country=request.user.monitor.country) if not request.user.is_superuser else qs
+        return qs_filter
```
Comment (kilemensi) [inline forms/admin.py]:
```suggestion
        return qs.filter(country=request.user.monitor.country) if not request.user.is_superuser else qs
```

**Review by kilemensi — COMMENTED** [review]
:+1: @ascii-dev 

--------------

When you just login, there is a small card that shows total number of stories coded so far:
Screenshot from 2020-09-25 14-16-57
In my local testing, this seem is still pulling everything and not limited to country.

File: gmmp/dashboard_modules.py [hunk 2]
```diff
@@ -100,27 +100,27 @@ def init_with_context(self, context):
         self.children = [
             {
                 "name": _("Newspapers"),
-                "count": NewspaperSheet.objects.count(),
+                "count": NewspaperSheet.objects.filter(country=context.request.user.monitor.country).count(),
```
Comment (kilemensi) [inline gmmp/dashboard_modules.py]: Super users should still be able to see everything @ascii-dev ... so probably introduce a method that takes `Model.objects` and add `filter` if user is not a super user similar to the `get_queryset`.

**Review by kilemensi — COMMENTED** [review]
:+1: @ascii-dev

File: gmmp/dashboard_modules.py (hunk 2 above)
Comment (ascii-dev) [reply gmmp/dashboard_modules.py]: Oh... True... I missed that... @kilemensi

File: gmmp/dashboard_modules.py [hunk 3]
```diff
@@ -96,31 +96,34 @@ class Submissions(DashboardModule):
     title = "Coded"
     template = "gmmp/dashboard_modules/submissions.html"
 
+    def get_count(self, model_, user):
```
Comment (kilemensi) [inline gmmp/dashboard_modules.py]: Isn't this for "internal" use only & hence should start with `_` i.e. `_get_count`?

**Review by kilemensi — APPROVED** [review]
:rocket:

**Pull request merged**